zowex/src/main.rs

```diff
@@ -241,18 +239,18 @@ fn arg_vec_to_string(arg_vec: &Vec<String>) -> String {
          * enclosed in double quotes when it is placed into a single argument
          * string.
          */
-        if next_arg.contains(' ') || next_arg.len() == 0 {
+        if next_arg.contains(' ') || next_arg.is_empty() {
             arg_string.push('"');
             arg_string.push_str(next_arg);
             arg_string.push('"');
         } else {
             arg_string.push_str(next_arg);
         }
 
-        arg_count = arg_count + 1;
+        arg_count += 1;
     }
 
-    return arg_string;
+    arg_string
```

Comment:
I am not a fan of the practice of NOT having a 'return' statement and just placing a variable name as the last statement in a function. But if that is something which the Rust linter complains about, we should probably conform.

Reply:
The category of this linting problem is "style:warn". We could disable this specific warning, or perhaps all "style" warnings if we don't care about how "idiomatic" our Rust code is? 😛 
| Category              | Description                                                                         | Default level |
| --------------------- | ----------------------------------------------------------------------------------- | ------------- |
| `clippy::all`         | all lints that are on by default (correctness, suspicious, style, complexity, perf) | **warn/deny** |
| `clippy::correctness` | code that is outright wrong or useless                                              | **deny**      |
| `clippy::suspicious`  | code that is most likely wrong or useless                                           | **warn**      |
| `clippy::style`       | code that should be written in a more idiomatic way                                 | **warn**      |
| `clippy::complexity`  | code that does something simple but in a complex way                                | **warn**      |
| `clippy::perf`        | code that can be written to run faster                                              | **warn**      |
| `clippy::pedantic`    | lints which are rather strict or have occasional false positives                    | allow         |
| `clippy::nursery`     | new lints that are still under development                                          | allow         |
| `clippy::cargo`       | lints for the cargo manifest                                                        | allow         |